### ISSUE TITLE
Exchange pages scrolling up/down unless mouse scroll wheel button is held down

### ISSUE THREAD

**Issue by brekyrself** [bitshares/bitshares-ui#3687]
When on any exchange page/trading pair and the mouse cursor is located on the left window pain or the right window my markets / find markets section, simply moving the mouse up or down scrolls the page.  Moving the mouse down makes the page scroll up, moving the mouse up makes the page scroll down.  Only solution is to hold the mouse scroll wheel button which allows operation back to normal behavior.

**To Reproduce**
1. Go to 'Any exchange pair'
2. hover over the left pain where the charts are & buy/sell data is located.
3. move mouse up or down

**Expected behavior**
Page should not scroll by simply moving the mouse up or down without dragging the scroll bars or using the scroll wheel.

**Screenshots**
I![mouse movement](https://github.com/bitshares/bitshares-ui/assets/15274760/1cfaf7cf-8f63-424c-8359-bd3b52d17102)

**Desktop (please complete the following information):**
 - OS: Ubuntu 23.04 & 23.10
 - Reference wallet
 - Version 230907

**Comment by abitmore** [comment]
What browser?

**Comment by brekyrself** [comment]
> What browser? NVM, just noticed it's the downloaded version. So it might be Chromium.

Correct, downloaded version.  No problems using a chromium based browser (Brave) @ develop.bitshares.org.
